hilti/runtime/include/types/stream.h

```diff
@@ -85,8 +90,7 @@ class Chunk {
     using Array = std::pair<Size, std::array<Byte, SmallBufferSize>>;
     using Vector = std::vector<Byte>;
 
-    Chunk(Offset o, std::array<Byte, SmallBufferSize>&& d, Size n)
-        : _offset(o), _data(std::make_pair(n, std::move(d))) {}
+    Chunk(Offset o, std::array<Byte, SmallBufferSize> d, Size n) : _offset(o), _data(std::make_pair(n, d)) {}
```

Comment:
Why this change?

Reply:
This fixes a clang-tidy warning about an unnecessary `move` since the array here is trivially copyable. I'll move this to its own patch.